packages/component-library/src/components/form/mt-number-field/mt-number-field.vue

```diff
@@ -389,38 +386,38 @@ export default defineComponent({
 
 <style lang="scss">
 .mt-field {
-  &--controls {
-    background: $color-white;
+  &__controls {
+    background: var(--color-elevation-surface-raised);
     display: flex;
     flex-direction: column;
-    justify-content: space-evenly;
+    justify-content: center;
+    row-gap: 8px;
     align-items: center;
     width: 42px;
 
-    &.disabled {
-      background: $color-gray-100;
-      border-color: $color-gray-300;
+    &--disabled {
+      background: var(--color-background-primary-disabled);
       cursor: default !important;
 
       .mt-icon:hover {
         cursor: default !important;
-        color: $color-gray-500;
       }
     }
 
-    &.error {
-      background: $color-crimson-50;
+    &--has-error {
+      background: var(--color-background-critical-dark);
     }
 
     .mt-icon {
       cursor: pointer;
-      color: $color-gray-500;
+      color: var(--color-icon-primary-default);
 
-      &--toggled {
-        color: $color-gray-900;
+      & svg {
+        width: 12px !important;
+        height: 6px !important;
       }
 
-      &:hover {
+      &--toggled {
         color: $color-gray-900;
```

Comment:
```suggestion
        color: var(--color-icon-primary-default);
```

Reply:
I remove pressed state for the spinners.